installer/cli/microk8s.py

```diff
@@ -104,6 +104,7 @@ def install(args) -> None:
     parser.add_argument('--cpu', default=definitions.DEFAULT_CORES, type=int)
     parser.add_argument('--mem', default=definitions.DEFAULT_MEMORY, type=int)
     parser.add_argument('--disk', default=definitions.DEFAULT_DISK, type=int)
+    parser.add_argument('--assume-yes', action='store_true', default=definitions.DEFAULT_ASSUME)
```

Comment:
Two comments:
1. This new arg should be added in `_show_install_help()` also.
2. There really should be a `-y` shortcut for this also, since `-y` is kind of the universally used shorthand for "run this command and automatically say 'Yes' anywhere you would have prompted for my opinion."